### ISSUE TITLE
fun models take extremely long time

### ISSUE THREAD

**Issue by john2stai** [kijai/ComfyUI-CogVideoXWrapper#279]
Is there any particular reason why the fun models are barely useable in terms of speed? I've been trying fun 1.1 5b inpaint and fun 5b inpaint models with or without tora for image to video, but after several attempts, all I got were 20+ hours estimated time for generating a 512*384, 20 steps, cfg 8 video. 
Could this be a platform-specific issue? I am on mac.

**Comment by KrakeyMTL** [comment]
That seems far far too low. I tested the Fun models yesterday and they do work here in less than 15min on my servers - BUT I'm not using inpainting ones. Maybe that? I would need to try an inpainting setup and I don't currently use that.

Also mac's yes...depending on your chip this will be rough a bit. 

Mac specs? This will help us.

**Comment by john2stai** [comment]
macbook pro m3 pro 36gb ram

**Comment by john2stai** [comment]
> That seems far far too low. I tested the Fun models yesterday and they do work here in less than 15min on my servers - BUT I'm not using inpainting ones. Maybe that? I would need to try an inpainting setup and I don't currently use that.
> 
> Also mac's yes...depending on your chip this will be rough a bit.
> 
> Mac specs? This will help us.

macbook pro m3 pro 36gb ram

**Comment by scottmudge** [comment]
5b models are going to be quite slow on Mac's, their GPU performance is quite poor compared to a dedicated GPU. The tradeoff is you have more (unified) RAM than most GPUs.

**Comment by KrakeyMTL** [comment]
I would agree, I very rarely play with a mac in order to test but will have access to a M2 machine soon actually. Apple isn't very keen on dedicated gpu's sadly. Maybe with this years insanity they will put more thought into it!

Not to derail the topic - the only 2 useful tidbits I found were this for mac ai gen stuff (you probably already have seen these in your research apologies) 
-jibber jabber confirming things do in fact work and very fast when they have a unique config so we do know people have it working from M1's to M3's -> https://www.reddit.com/r/StableDiffusion/comments/17ol8bg/stable_diffusion_on_macbook_pro_m3/

- alt software  https://apps.apple.com/us/app/draw-things-ai-generation/id6444050820  to try and "see" if it's comfyui or the machine in a way. By all accounts this is supposed to be the auto1111 of the mac world and well upkept to boot, how I never heard of this before is a mystery (this is what I'll be trying on the M2 soon)

So from what I gather this very much could be comfyui and it being wonky needing to be hammered out with macs. If the above thingy gens on your M3 in less than 30 seconds we know it has the power - then it's just to figure out the configs.

This is about the limit of my current support for mac bud, sorry I couldn't help further. In a week or two maybe! I'll update if this thread is still milling around.
